### DISCUSSION TITLE
Restricted admin role or ability to customize admin role

### DISCUSSION THREAD

**Discussion by vanessajbell** [directus/directus#4367] [Feature Requests]
Right now, admin users get access to everything in Directus. Ideally there would be a way to allow access to only certain settings. E.g. a role that has access to the Directus settings (admin panel) and can create/update schemas but cannot add or archive users.

**Comment by philleepflorence** [comment]
I can see 2 Default Administrator [Roles](https://docs.directus.io/concepts/roles/) (Pre-built Administrator Roles):

1. [Project](https://docs.directus.io/concepts/projects/) Administrator - The current administrator role or Super Administrator.
2. [Application](https://docs.directus.io/concepts/application/) Administrator - Same as above without any user administrative privileges.

Is this closely related to what you mean?
Do you see other default or pre-built administrator roles?

**Reply by vanessajbell** [reply]
Application administrator would be a step in the right direction, but for the role we have in mind, the user would only be able to add and edit data models. They wouldn't have access to roles & permissions, project settings, etc. So perhaps what we're looking for is the ability to customize an Administrator role.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
